cmd/antrea-agent/agent.go

```diff
@@ -133,6 +139,8 @@ func run(o *Options) error {
 
 	go agentMonitor.Run(stopCh)
 
+	antctl.StartServer(antctlServerOpt, antctl.CommandList, agentMonitor, nil, stopCh)
```

Comment:
Why here we don't introduce a new antctl stop channel to graceful exit?

Reply:
It did do like you say, but folks would like to have a consistent solution about the stop channel.